src/lib/components/workflow/search-attribute-input/index.svelte

```diff
@@ -22,9 +23,9 @@
   export let onRemove: (attribute: string) => void;
 
   $: type = searchAttributes[attribute.attribute];
-  $: searchAttributesOptions = [...Object.entries(searchAttributes)]
-    .map(([key, value]) => ({ label: key, value: key, type: value }))
-    .filter(({ type }) => type !== 'KeywordList');
+  $: searchAttributesOptions = [...Object.entries(searchAttributes)].map(
```

Comment:
I don't super care, but I might pull this out and test it.

Reply:
[Updated this to use a derived store](https://github.com/temporalio/ui/pull/2420/commits/69766cf78b1a7cfdebc87999969faa877a0f0363) for now.